src/test/java/com/cdancy/bitbucket/rest/features/PullRequestApiLiveTest.java

```diff
@@ -99,7 +99,7 @@ public void init() {
     private void createPullRequest() {
         final String randomChars = TestUtilities.randomString();
         final ProjectKey proj = ProjectKey.create(project);
-        final MinimalRepository repository = MinimalRepository.create(repo, null, proj);
+        final MinimalRepository repository = MinimalRepository.create(42, repo, null, proj);
```

Comment:
Why the number `42` everywhere? Meaning of life?

Reply:
It's the [ultimate answer to life, the universe, and everything](https://en.wikipedia.org/wiki/42_(number)#The_Hitchhiker's_Guide_to_the_Galaxy). I could make it random though?